w2w/w2w.py

```diff
@@ -217,6 +229,38 @@ def from_argparse(cls, args: argparse.Namespace) -> 'Info':
         )
 
 
+def _get_lcz_band(info: Info, args: argparse.Namespace) -> int:
+
+    # Read the file
+    lcz = rxr.open_rasterio(info.src_file)
+
+    # Check if 'lczFilter' is part of attributes. If so, band = 1
+    if 'long_name' in lcz.attrs and 'lczFilter' in lcz.attrs['long_name']:
+        LCZ_BAND = 1
+        print(
+            '> Seems you are using a LCZ map produced by '
+            'https://lcz-generator.rub.de/ \n'
+            '> I therefor use the Gaussian filtered default '
+            "'LczFilter' layer"
+        )
+    # If no argument provided, LCZ_BAND defaults to 0
+    elif not args.LCZ_BAND:
+        LCZ_BAND = 0
+        print(
+            '> -l (--lcz_band) argument is not set. \n'
+            '> Defaulting to using the first layer of the LCZ GeoTIFF'
+        )
```

Comment:
since we set the default to `0`, this case is impossible.

Reply:
I think this branch can be removed entirely. Since if no argument is set, the argument is `0` and this way equal to when it is set.

Reply:
or could be the last one, that a user knows the default (0) is used?